cocotb/__init__.py

```diff
@@ -106,8 +107,36 @@ def _reopen_stream_with_buffering(stream_name):
 plusargs = {}
 """A dictionary of "plusargs" handed to the simulation."""
 
-# To save typing provide an alias to scheduler.add
-fork = scheduler.add
+
+def current_test() -> cocotb.decorators.RunningTest:
```

Comment:
My hunch is that we don't want to make this public until we have a good use case.

Reply:
The only use case I can think of is aborting the test, assuming we eventually move to test just being a `RunningTask`. Will remove.